src/main/java/frc/robot/commands/CameraDriveCommand.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import java.lang.Math;
+
+public class CameraDriveCommand extends CommandBase {
+  /** Creates a new CameraDrive. */
+  Drivetrain drive;
+  Timer diveTimeout = new Timer();
+
+    double ballX;
+    double ballY;
+
+  double moveXValue;
+  double moveYValue;
+  public CameraDriveCommand(Drivetrain drivetrain) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.drive = drivetrain;
+    addRequirements(drive);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+
+double XDeadzone = 6.5; // inches
+double YDeadzone = 4;
+double forwardDriveSpeed = 0.6;
```

Comment:
If this is a constant then it should be `static final` and in `ALL_CAPS`. Class-level variables and constants should also be at the top, below the class declaration, not under the constructor.